src/status_im2/common/scroll_page/view.cljs

```diff
@@ -80,19 +65,28 @@
           :mid-section             {:type            :text-with-description
                                     :main-text       (when (>= @scroll-height scroll-position-1) name)
                                     :description-img (when (>= @scroll-height scroll-position-1) icon)}
-          :right-section-buttons   (:right-section-buttons page-nav)
+          :right-section-buttons   page-nav-right-section-buttons
           :left-section            {:icon                  :i/close
                                     :icon-background-color (icon-color)
                                     :on-press              #(rf/dispatch [:navigate-back])}}]
-        (when sticky-header [sticky-header @scroll-height])]
+        sticky-header]
        [rn/scroll-view
-        {:style (style/scroll-view-container (diff-with-max-min @scroll-height 16 0))
+        {:style                           (style/scroll-view-container (diff-with-max-min @scroll-height
+                                                                                          16
+                                                                                          0))
          :shows-vertical-scroll-indicator false
-         :scroll-event-throttle 4
-         :on-scroll #(swap! scroll-height (fn [] (int (oops/oget % "nativeEvent.contentOffset.y"))))}
+         :scroll-event-throttle           4
+         :on-scroll                       #(do
```

Comment:
https://github.com/bbatsov/clojure-style-guide#no-multiple-forms-in-function-literals

From the guidelines, let's avoid multiple forms within `#()` as it hurts readability. I know some of us might disagree, but it's just easier if we follow the guide. I would love to have this reinforced by zprint, but I haven't bothered looking.

So replace the `#()` with `(fn [^js event])`. Some might say the type hint is unnecessary because we're using `oops/oget`, so it's up to you. I usually prefer to explicitly type hint native objects in function parameters.